sigmastate/src/main/scala/sigmastate/utxo/transformers.scala

```diff
@@ -22,9 +26,14 @@ case class MapCollection[IV <: SType, OV <: SType](
                                                     mapper: Value[SFunc])
   extends Transformer[SCollection[IV], SCollection[OV]] {
   override def companion = MapCollection
-  implicit def tOV = mapper.asValue[OV].tpe
   override val tpe = SCollection[OV](mapper.tpe.tRange.asInstanceOf[OV])
   override val opType = SCollection.MapMethod.stype.asFunc
+  protected final override def eval(E: ErgoTreeEvaluator, env: DataEnv): Any = {
```

Comment:
ScalaDoc for this and all the other touched methods

Reply:
done